File: scala/src/main/scala/io/swagger/client/model/QueueMessage.scala
package io.swagger.client.model

import org.joda.time.DateTime



case class QueueMessage (
  ACL: ACL,
  _type: String,
  expires: DateTime,
  id: String,
  delay: DateTime,
  timeout: Long,
  modifiedColumns: List[String],
  isModified: Boolean,
  message: String)
  

File: typescript-angular/API/Client/QueueMessage.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface QueueMessage {

        "aCL"?: ACL;

        "type"?: string;

        "expires"?: Date;

        "id"?: string;

        "delay"?: Date;

        "timeout"?: number;

        "modifiedColumns"?: Array<string>;

        "isModified"?: boolean;

        "message"?: string;
    }

}
